config/config.js

```diff
@@ -56,9 +56,9 @@ var validateEnvironmentVariable = function() {
     console.log();
     if (!environmentFiles.length) {
         if (process.env.NODE_ENV) {
-            console.error(chalk.red('No configuration file found for "' + process.env.NODE_ENV + '" environment using development instead'));
+            console.error(chalk.red('+ Error: No configuration file found for "' + process.env.NODE_ENV + '" environment using development instead'));
```

Comment:
To distinguish the rest of the startup text information on the screen from actual errors. I couldn't find any other better method for that. I can remove, it's just styling on the console.